EventGenerator/src/PrimaryProtonGun_module.cc

```diff
@@ -1,6 +1,8 @@
 
 /*
-  This is a Replicated Module.
+  This was a Replicated Module and was reverted to Legacy on Mar 27, 2002
```

Comment:
Typo here. You probably want this to say Mar 27, 2022.

Reply:
Thanks.  It's fixed.